spec/models/mixins/supports_feature_mixin_spec.rb

```diff
@@ -43,9 +43,9 @@ def initialize(values = {})
 
   describe ".supports_feature?" do
     it "defines supports on the class" do
-      expect(test_class.supports?(:std_accept)).to be_truthy
-      expect(test_class.supports?(:module_accept)).to be_truthy
-      expect(test_class.supports?(:std_denial)).to be_falsey
+      expect(test_class.supports_std_accept?).to be_truthy
+      expect(test_class.supports_module_accept?).to be_truthy
+      expect(test_class.supports_std_denial?).to be_falsey
```

Comment:
as mentioned in the commit message, these are tested below
but this tests specifically is testing the underscore methods. (so the test was written wrong)

ASIDE: I don't like these methods, and the goal of this whole PR is to remove these methods, but until then, we need to test these